### ISSUE TITLE
Trying to install the not Pro Version of DoTeewn (v.1.2.632) in unity 2021.1.1.16f1

### ISSUE THREAD

**Issue by FunCrafter1234** [Demigiant/dotween#508]
I have a somewhat new Project and I have installed the Package using the Package manager.
I did the install as explained and now I am stuck with multiple compiler errors pointing to missing types/namespaces for example:

Assets\Plugins\Demigiant\DOTween\Modules\DOTweenModuleEPOOutline.cs(5,7): error CS0246: The type or namespace name 'EPOOutline' could not be found (are you missing a using directive or an assembly reference?

Assets\Plugins\Demigiant\DOTween\Modules\DOTweenModuleEPOOutline.cs(14,39): error CS0246: The type or namespace name 'SerializedPass' could not be found (are you missing a using directive or an assembly reference?)

Assets\Plugins\Demigiant\DOTween\Modules\DOTweenModuleEPOOutline.cs(75,39): error CS0246: The type or namespace name 'Outlinable' could not be found (are you missing a using directive or an assembly reference?)

I tried to install it 4 times now with always the same result. 
I also tried to install it with the downloadable zip from the webside. Still no luck

Is this Package not supported with this Unity Version or am I just missing something? I could not find any help for this issue so it seems like I am more or less alone with this issue.

**Comment by pietroF42** [comment]
I have similar issues when I try install it using the package manager.. Interestingly though, if I remove the files/folders added by the package manager and instead copy and paste the content of [this](https://github.com/UnityTechnologies/open-project-1/tree/main/UOP1_Project/Assets/Addons/DOTween) it all works fine (make sure to restart the editor after that happens)

**Comment by Demigiant** [comment]
You shouldn't install via Package Manager but via the store. I've been trying for a while to get DOTween to work via Package Manager but it's complicated because of how I rewrite some files depending on the settings, and after failing numerous times to find a satisfying way I postponed it indefinitely. I mean to try a new idea in the next month, but I'm not sure how worth it would be, considering that the paid version would need the store anyway.

**Comment by pietroF42** [comment]
hmm, how do you mean?
when you go through the Unity store it still opens up the package manager to install the asset 🤔 

![image](https://user-images.githubusercontent.com/79974766/134167612-b3da582f-5583-41d5-8f16-d3ccec4f0a70.png)

**Comment by Demigiant** [comment]
Oh, sorry, yes, that's the right way, my bad. It's just that when referring to Package Manager people often refer to custom-built PM installs and not the store ones (so installs that come from the "Unity Registry" tab and not the correct "My Assets" tab).

About the problem the OP had, it's actually different and very straightforward, and I must've mixed this report with another considering my answer. The problem is the OP enabled a DOTween module for an external asset without having said external asset in their project (in the OP case, Easy Performant Outline). Just run DOTween's Setup again and disable any module under "External Assets" unless you do have those assets ;)

![image](https://user-images.githubusercontent.com/8322877/134323168-008bfab8-40c7-43eb-821a-dbf7df72ea64.png)

**Comment by FunCrafter1234** [comment]
I actually never have had that enabled. I read the instructions carefully and have checked everywhere, as I said I have installed this like 10 times and I always checked the external assets. Nothing was checked and it still brought up the error. After a few days of trying to fix it i gave up and only after a few weeks now I have opened the project again. The error now is gone without reinstalling again. Something else was going on there maybe it was just unity having its problems I have no idea but I can 100% say that the external assets were not checked.

**Comment by ShawnChiang886** [comment]
> Oh, sorry, yes, that's the right way, my bad. It's just that when referring to Package Manager people often refer to custom-built PM installs and not the store ones (so installs that come from the "Unity Registry" tab and not the correct "My Assets" tab).
> 
> About the problem the OP had, it's actually different and very straightforward, and I must've mixed this report with another considering my answer. The problem is the OP enabled a DOTween module for an external asset without having said external asset in their project (in the OP case, Easy Performant Outline). Just run DOTween's Setup again and disable any module under "External Assets" unless you do have those assets ;)
> 
> ![image](https://user-images.githubusercontent.com/8322877/134323168-008bfab8-40c7-43eb-821a-dbf7df72ea64.png)

ths!

**Comment by sadiqhusaini** [comment]
Dear @Demigiant team,

I hope this message finds you well. I am writing to bring to your attention an issue we are encountering on the iOS platform. Our current setup involves utilizing the latest versions of Dotween Pro and Easy Outline.

Specifically, we are experiencing difficulties and require your assistance in resolving this matter. It would be greatly appreciated if you could provide guidance or support to help us overcome this challenge.

Thank you for your attention to this matter, and we eagerly await your prompt response. It's the same error as mentioned in the main message.

**Comment by rtykholiz** [comment]
True, i disabled "External assets" and dotween works fine.
